torch/distributed/rpc/internal.py

```diff
@@ -158,10 +160,7 @@ def _start_record_function(exec_type, func_name, current_worker_name, dest_worke
         dest_worker_name (str): Name of the destination worker.
 
     Returns:
-        An instance of `torch.autograd._RecordFunction`. This function also has
-        the side effect of running the before callbacks that start the
-        profiling, though the user is responsible for running the appropriate
-        callbacks when the function to be profiled finishes.
+        An instance of `torch.autograd._RecordFunction`.
     """
     assert torch.autograd._profiler_enabled(), "Autograd profiler should be enabled."
     profile_key = "rpc_{}.{}({} -> {})".format(
```

Comment:
nit: Maybe something like this would be more readable `rpc_sync#foo.bar.baz(worker0->worker1)`. Using the # would help in delimiting rpc_sync and the actual package name. Otherwise something like rpc_sync.foo.bar.baz might be interpreted as a single package.